<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupServoAngles">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>501</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Servo Angles</string>
    </property>
    <widget class="QCustomPlot" name="plotServo" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>431</width>
       <height>297</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QScrollBar" name="horScrollServo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBPlotServo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupXYCoords">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>501</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>X-Y Graph</string>
    </property>
    <widget class="QCustomPlot" name="plotXY" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>431</width>
       <height>297</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QScrollBar" name="horScrollXY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBPlotXY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnConnPlate">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect Plate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpen3D">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Open 3D Simulation</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBMsg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>401</width>
      <height>111</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Some Messages Here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPortName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>/dev/ttyACM0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputBaudRate">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>9600</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPortName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBaudRate">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud Rate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisconnect">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kpLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kiLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kdLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="kpLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kp</string>
    </property>
   </widget>
   <widget class="QLabel" name="kiLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ki</string>
    </property>
   </widget>
   <widget class="QLabel" name="kdLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kd</string>
    </property>
   </widget>
   <widget class="QLabel" name="gtuLogo">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCSE395">
    <property name="title">
     <string>Group 6</string>
    </property>
   </widget>
   <addaction name="menuCSE395"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
